### PR TITLE
Add show TASTy command

### PR DESCRIPTION
**Pull request by kpodsiad**
Changes required by https://github.com/scalameta/metals/pull/3063

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -114,7 +115,7 @@ export async function activate(context: ExtensionContext) {
           workspace.getConfiguration("metals").get("javaHome")
         );
         return fetchAndLaunchMetals(context, javaHome);
-      } catch (err) {
+      } catch (err: any) {
```
Comment (tgodzik) [inline src/extension.ts:118]: This should not be needed:
```suggestion
      } catch (err) {
```

File: package.json [hunk 2]
```diff
@@ -620,4 +629,4 @@
   "extensionDependencies": [
     "scala-lang.scala"
   ]
-}
+}
```
Comment (tgodzik) [inline package.json:632]: Please run `yarn format`

**Review by tgodzik — CHANGES_REQUESTED** [review]
Looks good! Just a few smaller issues and you need to rebase.

**Review by tgodzik — APPROVED** [review]
LGTM! Let's wait for the next release.

**Review by gabro — APPROVED** [review]

**Pull request merged**